crates/nargo/src/cli/prove_cmd.rs

```diff
@@ -60,11 +66,13 @@ pub(crate) fn run(args: ProveCommand, config: NargoConfig) -> Result<(), CliErro
         args.verify,
         args.show_ssa,
         args.allow_warnings,
+        args.inputs,
```

Comment:
The only arg not passed here is `circuit_name`. Let's just pass the whole args object?